cfgov/jinja2/v1/_includes/organisms/post-preview.html

```diff
@@ -44,7 +44,8 @@
 {% from 'macros/util/format/url.html' import location_image_url as location_image_url %}
 
 {% if page %}
-{% macro render(post, type, form_id) %}
+{% macro render(post, type='', form_id=0, url='') %}
+    {% set url = url or request.path %}
```

Comment:
It's a bit more efficient to just pass in the desired URL instead of the entire page object.

Also, `get_page_state_url()` doesn't exist in this PR. It's in the other one. https://github.com/cfpb/cfgov-refresh/pull/1413/files#diff-4adb797bae6921a1dc169c494a5d776fR23